### ISSUE TITLE
`@apply` doesn't respect order of sibling declarations within same rule

### ISSUE THREAD

**Issue by sebszocinski** [tailwindlabs/tailwindcss-jit#35]
There are times where you still want to use good old CSS and previously I would add these under my `@apply` statements, and they'd be added to the bottom of the built css file and therefor override tailwind, but with jit they are being added to the top. eg:

**Disclaimer: I know I can do this manual css using tailwinds `bg-opacity-50`  it's just an example...**

### app.sass
```
.some-div
  @apply bg-blue flex
  background: rgba(0,0,0,0.5)
```
### app.css (Built)
```
.some-div
  background-color: blue
  display: flex
  background-color: rgba(0,0,0,0.5)
```

### app.sass
```
.some-div
  @apply bg-blue flex
  background: rgba(0,0,0,0.5)
```
### app.css (Built)
```
.some-div
  background-color: rgba(0,0,0,0.5)
  background-color: blue
  display: flex
```

**Comment by adamwathan** [comment]
Hey thanks for reporting! We know about this issue and will work on it soon 👍🏻 Updating the title just to make it easier to remember what this one is about when I see it in the list.

**Comment by nash403** [comment]
Hi 👋 I ran into the same issue with `@screen` directive definition getting hoisted before other `@apply` definitions.

eg:
my definition
```
.some-div
  @apply bg-blue flex
  @screen md {
    width: 30rem;
  }
```

built:

**Comment by adamwathan** [comment]
Should be fixed in v0.1.9 (hopefully! 🙈)
